### PR TITLE
Add membership checklist template to Onboarding

### PR DESCRIPTION
**Pull request by bnb**
I've had this as a GitHub quick response for over a year now, didn't realize it apparently wasn't documented. Document all the things!

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -160,6 +160,16 @@ the CommComm restore them to active status.
...
 
+This template can be used when onboarding new members:
+
+```
+CommComm Membership Checklist: 
+
+- [ ] Invite to CommComm GitHub Team (and Node.js GitHub org, if not already member)
+- [ ] Submit PR to add to the [CommComm email alias](https://github.com/nodejs/email/blob/master/iojs.org/aliases.json#L96)
+- [ ] Submit PR to add to the (CommComm README.md](https://github.com/nodejs/community-committee/blob/master/README.md#community-committee-members)
+```
```
Comment (mhdawson) [inline GOVERNANCE.md]: We also need to add them to the invited list in the templates for https://github.com/nodejs/create-node-meeting-artifacts

**Review by keywordnew — APPROVED** [review]

**Review by WaleedAshraf — APPROVED** [review]

**Review by amiller-gh — APPROVED** [review]
Love this – for completeness' sake I'll ask: Do we also need off-boarding instructions? (@bnb)

**Review by codeekage — APPROVED** [review]
LGTM

**Comment by chowdhurian** [comment]
Good call, Adam. #392 opened.

Anyone familiar with, or having input to, the offboarding process (@bnb 👀) please comment!

File: GOVERNANCE.md [hunk 2]
```diff
@@ -160,6 +160,17 @@ the CommComm restore them to active status.
 All collaborators added as CommComm members should be on-boarded in a timely manner
 and be given write access to the repository.
 
+This template can be used when onboarding new members:
+
+```
+CommComm Membership Checklist: 
+
+- [ ] Invite to CommComm GitHub Team (and Node.js GitHub org, if not already member)
+- [ ] Submit PR to add to the [CommComm email alias](https://github.com/nodejs/email/blob/master/iojs.org/aliases.json#L96)
```
Comment (thefourtheye) [inline GOVERNANCE.md]: The line number 96 in aliases.json doesn't correspond to commcomm. Its better to drop the line number from this.

**Review by thefourtheye — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by bnb** [comment]
Since this is uncontroversial I'll merge it 👍

**Pull request merged**